torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1970,6 +1970,17 @@ Tensor elu_double_backward(
     }
 }
 
+Tensor sinc_backward(const Tensor& grad, const Tensor& self) {
+  Tensor self_scaled = at::ones_like(self);
+  Tensor scale_v;
+  Tensor pi = wrapped_scalar_tensor(Scalar(M_PI));
+  scale_v = grad *
+      ((pi * self * (pi * self).cos() - (pi * self).sin()) / (pi * self * self))
```

Comment:
now that you moved this to an out of place function, you can pre-compute `pi * self` on the line above and re-use it here. That will improve speed quite a bit.